site/src/components/address-input/possibly-hidden-select.tsx

```diff
@@ -0,0 +1,37 @@
+import {Component, h, Prop} from "@stencil/core";
+
+@Component({
+   tag: "possibly-hidden-select",
+   shadow: false
+})
+export class PossiblyHiddenSelect {
```

Comment:
@nexussays got a warning when I tried to put this component in the same file as `address-input`, so it's here separately.

Reply:
That works. It's probably easier to just make a functional component, but you've already done the work so no reason to change it 👍